src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/YourSingleApprenticeWithTrainingProviderStatus.cshtml

```diff
@@ -0,0 +1,68 @@
+﻿@using SFA.DAS.EmployerAccounts.Web.Extensions
+@model SFA.DAS.EmployerAccounts.Web.ViewModels.CohortV2ViewModel
+
+<section class="dashboard-section">
+    <h2 class="section-heading heading-large" id="call-to-action-your-apprentice-status">
+        Your apprentice
+    </h2>
+
+    <table class="responsive">
+        <tr>
+            <th scope="row" class="tw-35">
+                Name
+            </th>
+            <td class="tw-65" id="apprentice-name">             
+                @(Model?.Apprenticeships?.First().ApprenticeshipFullName)
+            </td>
+        </tr>
+        <tr>
+            <th scope="row">
+                Training Provider
+            </th>
+            <td id="provider-name">
+                @(Model?.Apprenticeships?.First().TrainingProvider?.Name)
+            </td>
+        </tr>
+        <tr>
+            <th scope="row">
+                Course
+            </th>
+            <td id="course-name">
+                @(Model?.Apprenticeships?.First().CourseName)
+            </td>
+        </tr>
+        <tr>
+            <th scope="row">
+                Status
+            </th>
+            <td id="with-training-provider" class="govuk-tag govuk-tag--inactive">
+                WITH TRAINING PROVIDER
+            </td>
+        </tr>
+        @if (Model.Apprenticeships.First().CourseEndDate.HasValue)
+        {
+            <tr>
+                <th scope="row">
+                    Start month
+                </th>
+                <td id="course-start-date">
+                    @(Model?.Apprenticeships?.First().CourseStartDate.Value.ToString("MMMM yyyy"))                    
+                </td>
+            </tr>
+            <tr>
+                <th scope="row">
+                    Finish month
+                </th>
+                <td id="course-end-date">
+                    @(Model?.Apprenticeships?.First().CourseEndDate.Value.ToString("MMMM yyyy"))                    
```

Comment:
Please add CourseEndDateText property to the ApprenticeshipViewModel and set the format in the mapper. Then it's just in one place.